content/general/concepts/what-is-UUID/what-is-UUID.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'What is UUID?'
+Description: 'UUID is a universally unique identifier. It can be generated on any computer without the use of a centralized authority. It's a string of 128 bits that is supposed to be unique with little to no chance of clashing with any other UUID produced by anybody else in the world. UUIDs are usually used in database tables as unique keys. They are useful any time you need an ID that needs to be unique in an application or system.'
+Subjects:
+	- 'Computer Science'
+  - 'Web Development'
+Tags:
+	- 'Computer Science'
+  - 'Web Development'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+UUID is a universally unique identifier. It can be generated on any computer without the use of a centralized authority.
+It's a string of 128 bits that is supposed to be unique with little to no chance of clashing with any other UUID produced by anybody else in the world.
```

Comment:
If these sentences were intended to be on their own separate lines, let's add an empty line between 15 and 16. We never want to have paragraphs scrunched next to other paragraphs, code blocks, headers, etc., and vice versa.